openstack/castellum/templates/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: castellum-secret
+data:
+  service_user_password: {{ required ".Values.castellum.service_user.password" .Values.castellum.service_user.password | b64enc }}
```

Comment:
The `required` will never trigger because the default is the non-empty string `DEFINED_IN_VALUES_FILE`. Since we did without the `required` calls before, I think we'll be good without going forward as well.